modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -436,7 +436,14 @@ public ObjectDefinition putObjectDefinition(
 				titleServiceBuilderObjectField.getObjectFieldId();
 		}
 
-		if (serviceBuilderObjectDefinition.isUnmodifiableSystemObject()) {
+		boolean systemObject = serviceBuilderObjectDefinition.isSystem();
```

Comment:
@mateussgn can you rename this variable for unmodifiableSystemObject. I'm still not sure what Brian will prefer, but let's see.
Please, do the same with the other ones.

Reply:
I think it was here we misunderstood. I was mentioning about this variable: `systemObject`

systemObject -> unmodifiableSystemObject

and not the method :S